Marharyta_Miliuk/Module 3 - Java/Week 1/TAM_JAVA/src/hometask1/week1/module3/tam/InputOutput.java

```diff
@@ -0,0 +1,60 @@
+package hometask1.week1.module3.tam;
+import java.util.*;
+
+public class InputOutput {
+	
+	private static String firstMessage = "Please, enter first number";
+	private static String secondMessage = "Please, enter second number";
```

Comment:
If you want to declare constant you need to write "final". Also style for constants: FIRST_MESSAGE. Please correct this